### DISCUSSION TITLE
CSS variables are more intense than they appear

### DISCUSSION THREAD

**Discussion by WynieCronje** [taiga-family/taiga-ui#136] [General]
After reading this:
https://www.w3schools.com/css/css3_variables.asp

I played around with one demo: (toggle body classes between : '' , 'blue' , 'red')
https://www.w3schools.com/code/tryit.asp?filename=GMWW7XHSLUOY

😮  Now that's cool

**Comment by waterplea** [comment]
Not sure what was the intended message :) If it's just a discussion on CSS vars then yeah, they are pretty cool. They do not work in IE but there's a working ponyfill:
https://github.com/jhildenbiddle/css-vars-ponyfill

We used to run it but now we are dropping IE support :)

**Reply by WynieCronje** [reply]
Yeah :).. i just read up about them today. And kinda got excited.

If someone  has some cool tricks with them. We might be able to showcase them here 😄

**Reply by loxy** [reply]
Have a look at https://codyhouse.co/ds/docs/framework. They do cool things with CSS Vars

**Reply by waterplea** [reply]
Here's a couple of CSS vars tweets from me :) Second one is used to style charts in Taiga

https://twitter.com/Waterplea/status/1283755056731508742

https://twitter.com/Waterplea/status/1399637643537813504
